src/server.js

```diff
@@ -0,0 +1,24 @@
+const {checkWatcher, startNPMWatcher} = require('./watchNPM')
+const redis = require('redis')
+const amqp = require('amqplib')
+const logger = require('~/src/logging').logger(module)
+const config = require('~/src/config')
```

Comment:
To be able to use these kinds of path, you will need to add the `require-self-ref` dependency to the project and place a `require('require-self-ref')` at the top of whatever file is the entry point of your app (this file).